src/libraries/System.Private.Xml/tests/Writers/XmlWriterApi/TCFullEndElement.cs

```diff
@@ -6352,12 +6352,12 @@ public partial class TCClose
                 [XmlWriterInlineData]
                 public void var_1(XmlWriterUtils utils)
                 {
-                    using (XmlWriter writer = utils.CreateWriter())
-                    {
-                        writer.WriteStartElement("Root");
-                        writer.WriteStartElement("Nesting");
-                        writer.WriteStartElement("SomeDeep");
-                    }
+                    XmlWriter writer = utils.CreateWriter();
+                    writer.WriteStartElement("Root");
+                    writer.WriteStartElement("Nesting");
+                    writer.WriteStartElement("SomeDeep");
+                    writer.Close();
```

Comment:
why are we using Close over Dispose here? If it's just about flattening I'd go ahead with `using XmlWriter writer = ...`, if there is something wrong with Dispose it's most likely a bug

Reply:
It's because this test claims to test Close, per its comment. The next test tests Dispose. Some copy paste error of the past

Reply:
simpler than I thought I guess :-)